prow/gcsupload/run.go

```diff
@@ -90,12 +92,16 @@ func (o Options) assembleTargets(spec *downwardapi.JobSpec, extra map[string]gcs
 		if info.IsDir() {
 			gatherArtifacts(item, gcsPath, info.Name(), uploadTargets)
 		} else {
-			destination := path.Join(gcsPath, info.Name())
+			name := info.Name()
```

Comment:
We don't seem to use this more than once? Why split it out?

Reply:
> We don't seem to use this more than once? Why split it out?

Fixed with e76f26879 -> 0b7effa0f.